README.md

```diff
@@ -37,7 +37,7 @@ Using [npm](https://www.npmjs.com/):
 
 Then with a module bundler like [webpack](https://webpack.github.io/) that supports either CommonJS or ES2015 modules, use as you would anything else:
 
-```js
+```jsx
```

Comment:
I thought about that a little. The JSX syntax highlighting is slightly different for vanilla JavaScript, though:

```js
// js
const foo = { bar: 'baz' };
```

```jsx
// jsx
const foo = { bar: 'baz' };
```

Given that, it seemed like consistency in syntax highlighting was preferable to strict correctness.

And it let me do everything with a regex.